src/ctia/stores/es/query.clj

```diff
@@ -96,9 +96,22 @@ Returns a map where key is path to a field, and value - path to the nested text
     :props {s/Keyword s/Any}
     :index s/Any
     :conn s/Any
-    :services s/Any}))
+    :services s/Any
+    :searchable-fields (s/maybe #{s/Keyword})}))
 
-(s/defn rename-search-fields :- (s/maybe {s/Keyword [s/Str]})
+(s/defn enforce-search-fields :- (s/maybe [s/Str])
```

Comment:
I would unit test this function.

Reply:
https://github.com/threatgrid/ctia/blob/move-enforce-search-fields/test/ctia/http/generative/fulltext_search_test.clj#L410

Reply:
>  I am a little bit surprise to not see any impact on refine-full-text-query-parts-test

Oh, you are right - I need to extend the test, taking into account that enforcing fields now happens inside this function. My bad. I'll add some test cases.